cli/loader_lib.c

```diff
@@ -244,22 +260,24 @@ __attribute__((constructor)) void jl_load_libjulia_internal(void) {
 #ifdef LIBSTDCXXPROBE
     void* handle;
     int do_probe = 1;
-    char* probevar = getenv("JULIA_LIBSTDCXXPROBE");
+    char* probevar = getenv("JULIA_PROBELIBSTDCXX");
     if (probevar) {
         if (strcmp(probevar, "1") == 0 || strcmp(probevar, "yes"))
             do_probe = 1;
         else if (strcmp(probevar, "0") == 0 || strcmp(probevar, "no"))
             do_probe = 0;
     }
     if (do_probe) {
-        char cxxpath[JL_PATH_MAX];
+        char* cxxpath = NULL;
```

Comment:
style nit: star leans right (as a reminder that it "belongs" to the expression on the right, semantically)
```suggestion
        char *cxxpath = NULL;
```